subprojects/gradle/impact-shared/src/main/java/com/avito/impact/configuration/BaseConfiguration.kt

```diff
@@ -46,19 +46,33 @@ abstract class BaseConfiguration(
     }
 
     fun allDependencies(includeSelf: Boolean = true): Set<BaseConfiguration> {
-        val dependencies = this.dependencies
-            .flatMap {
-                it.allDependencies(includeSelf = true)
```

Comment:
`allDependencies` calculated for each node recursively, but we need only for one node.